Makefile

```diff
@@ -11,7 +11,7 @@ test:
 	mvn clean test
 
 generate: install
-	bash scripts/generate.sh
+	bash scripts/generate.sh $(LANG)
```

Comment:
These changes aren't need as the env vars will be accessible by name in the scripts.